semgrep/semgrep/error.py

```diff
@@ -57,6 +57,7 @@ def to_dict(self) -> Dict[str, Any]:
         return {
             "type": self.__class__.__name__,
             "code": self.code,
+            "level": self.level.name.lower(),
```

Comment:
The `.lower()` shenanigans are to stay consistent with `ErrorWithSpan` and `--json` output :+1: